salt/transactional-update/10-update-rebootmgr-options.conf

```diff
@@ -0,0 +1,3 @@
+[Service]
+ExecStart=
+ExecStart=/usr/sbin/transactional-update cleanup dup salt
```

Comment:
I think @grahamhayes is right: the `transactional-updates` should be run at a higher frequency, but it should not have nasty side effects like creating multiple snapshots or downloading the same packages again and again... We should tackle these problems after 1.0.